cluster/gce/gci/master-helper.sh

```diff
@@ -36,15 +36,16 @@ function create-master-instance {
 
   write-master-env
   ensure-gci-metadata-files
-  create-master-instance-internal "${MASTER_NAME}" "${address}"
+  create-master-instance-internal "${master_name}" "${address}"
 }
 
 function replicate-master-instance() {
   local existing_master_zone="${1}"
   local existing_master_name="${2}"
   local existing_master_replicas="${3}"
 
-  local kube_env="$(get-metadata "${existing_master_zone}" "${existing_master_name}" kube-env)"
+  local kube_env
+  kube_env="$(get-metadata "${existing_master_zone}" "${existing_master_name}" kube-env)"
```

Comment:
on modifying:
```
In ./cluster/gce/gci/master-helper.sh line 47:
  local kube_env="$(get-metadata "${existing_master_zone}" "${existing_master_name}" kube-env)"
        ^------^ SC2155: Declare and assign separately to avoid masking return values.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2155
```